tests/standalone_tests/algorithms/exaflow/test_exaflow_algorithms.py

```diff
@@ -0,0 +1,57 @@
+import json
+import time
+
+import pytest
+import requests
+
+from tests.standalone_tests.conftest import ALGORITHMS_URL
+
+
+def get_parametrization_list_success_cases():
+    algorithm_name = "compute_average"
+    request_dict = {
+        "inputdata": {
+            "data_model": "dementia:0.1",
+            "datasets": ["edsd0"],
+            "y": [
+                "lefthippocampus",
+            ],
+            "filters": {
+                "condition": "AND",
+                "rules": [
+                    {
+                        "id": variable,
+                        "type": "string",
+                        "operator": "is_not_null",
+                        "value": None,
+                    }
+                    for variable in {"lefthippocampus"}
+                ],
+            },
+        },
+        "type": "exaflow",
+    }
+    return [(algorithm_name, request_dict)]
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "algorithm_name, request_dict",
+    get_parametrization_list_success_cases(),
+)
+def test_exaflow_algorithms(
```

Comment:
Rename the test and the package to something more specific?

`test_exaflow_dummy_algorithm`
`test_exaflow_compute_average`

???